.changeset/silver-ligers-grin.md

```diff
@@ -0,0 +1,9 @@
+---
+"@latticexyz/world": major
+---
+
+- Split `CoreSystem` into `AccessManagementSystem`, `BalanceTransferSystem`, `BatchCallSystem`, `CoreRegistrationSystem`
```

Comment:
Thoughts on `CoreRegistrationSystem` vs just `RegistrationSystem`?

Reply:
I guess I see `Core` as some aggregation prefix
agree that it's fine either way